uaclient/entitlements/base.py

```diff
@@ -71,8 +71,9 @@ def name(self) -> str:
     def valid_names(self) -> List[str]:
         """The list of names this entitlement may be called."""
         valid_names = [self.name]
-        if self.presentation_name != self.name:
-            valid_names.append(self.presentation_name)
+        if self.cfg.machine_token_file.is_present:
+            if self.presentation_name != self.name:
+                valid_names.append(self.presentation_name)
```

Comment:
This makes sense! I only think this check should go down in `presentation_name` since that is the function that directly looks at the machine_token_file.

So maybe something like this:
```python
    @property
    def presentation_name(self) -> str:
        """The user-facing name shown for this entitlement"""
        if self.cfg.machine_token_file.is_present:
            return (
                self.cfg.machine_token_file.entitlements.get(self.name, {})
                .get("entitlement", {})
                .get("affordances", {})
                .get("presentedAs", self.name)
            )
        else:
            return self.name
```